packages/larva-patterns/modules/story/story.vertical.js

```diff
@@ -0,0 +1,17 @@
+const clonedeep = require( 'lodash.clonedeep' );
+
+const story_vertical = clonedeep( require( './story.prototype' ) );
+
+story_vertical.story_grid_classes = story_vertical.story_grid_classes.replace(
```

Comment:
No `.replace` in this file. 

This is a method we would disallow in the linter.